src/videojs-hls.js

```diff
@@ -331,7 +331,7 @@ videojs.Hls.prototype.addCuesForMetadata_ = function(segmentInfo) {
       frame = metadata.frames[i];
       time = segmentOffset + ((metadata.pts - minPts) * 0.001);
       cue = new window.VTTCue(time, time, frame.value || frame.url || '');
```

Comment:
we probably should also see if we can use a `DataCue` instead of a `VTTCue` for this.